frontend/ui/data/koptoptions.lua

```diff
@@ -297,7 +298,7 @@ left to right or reverse, top to bottom or reverse.]]),
                 enabled_func = function(configurable)
                     return optionsutil.enableIfEquals(configurable, "text_wrap", 1)
                 end,
-                labels = {S.AUTO, S.LEFT, S.CENTER, S.RIGHT, S.JUSTIFY},
+                labels = {_("auto"), _("left"), _("center"), _("right"), _("justify")},
```

Comment:
Perhaps we should add contexts to some of these while we're in here. E.g., center can mean both "to center" and "middle" depending on context. My unofficial policy of not adding contexts unless a problem is known is about existing code, but if you're editing things anyway that's a slightly different matter.

```suggestion
                labels = {
                    C_("Alignment", "auto"),
                    C_("Alignment", "left"),
                    C_("Alignment", "center"),
                    C_("Alignment", "right"),
                    C_("Alignment", "justify"),
                },
```